src/components/ActivitiesList/ActivitiesList.tsx

```diff
@@ -15,27 +17,38 @@ interface ActivitiesListProps {
 }
 
 export default function Activities(props: ActivitiesListProps): ReactElement {
+  const [activitiesList, setActivitiesList] = useState(props);
+
   return (
-    <div className={styles.list__main}>
-      <h1 className={styles.list__h1}>Your Activities</h1>
-      <ul>
-        {props.activities.map((activity) => (
-          <li key={activity.id} className={styles.list__card}>
-            <h2>
-              <strong>Activity:</strong>: {activity.name}
-            </h2>
-            <p>
-              <strong>Description:</strong>: {activity.description}
-            </p>
-            <p>
-              <strong>Category:</strong>: {activity.category}
-            </p>
-            <p>
-              <strong>Vote Count:</strong>: {activity.voteCount}
-            </p>
-          </li>
-        ))}
-      </ul>
-    </div>
+    <ul>
+      {props.activities.map((activity) => (
+        <li key={activity.id} className={styles.list__card}>
+          <h2>Activity: {activity.name}</h2>
+          <p>Description: {activity.description}</p>
+          <p>Category: {activity.category}</p>
+          <p>Place: {activity.location}</p>
+          <p>When: {activity.date}</p>
+          <p>
+            Time: {activity.timeStart} - {activity.timeEnd}
+          </p>
+
+          <p>
+            <span className={styles.thumb_up}>
+              {activity.voteCountUp} &nbsp;
+              <button>
+                <FontAwesomeIcon icon={faThumbsUp} />
+              </button>
+            </span>
+            &nbsp; &nbsp;
+            <span className={styles.thumb_down}>
+              {activity.voteCountDown} &nbsp;
```

Comment:
We should talk to the backend team and find out whether there will be a separate number for upvotes and downvotes, or just a single number of votes which then gets increased and decreased. My understanding is that it will be the latter